markovs_household/data/timeseries.py

```diff
@@ -1,13 +1,18 @@
+from collections import namedtuple
 from dataclasses import dataclass
-from typing import Dict
+from datetime import timedelta
+
+TimeSeriesEntry = namedtuple("TimeSeriesEntry", "time, value")

```

Comment:
I guess this is fine for a quick test implementation, but then shouldn't we rather define a proper data class with typed fields?

Also, we should probably specify what unit the `time` and `value` fields have?